### PR TITLE
Migrated nux.test.js to Playwright

### PR DESCRIPTION
**Pull request by sandeshjangam**
## What?
Based on https://github.com/WordPress/gutenberg/pull/38570, part of https://github.com/WordPress/gutenberg/issues/38851. New User Experience (nux)  [nux.test.js](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/specs/editor/various/nux.test.js) to its Playwright counterpart.

Note - This PR is dependent on https://github.com/WordPress/gutenberg/pull/39748 PR. This PR or the required "create-post" utils file is required.

## Why?
[Migration to Playwright](https://make.wordpress.org/core/2022/03/23/migrating-wordpress-e2e-tests-to-playwright/)

## How?
[Migration to Playwright](https://make.wordpress.org/core/2022/03/23/migrating-wordpress-e2e-tests-to-playwright/)

## Testing Instructions
npm run test-e2e:playwright -- test/e2e/specs/editor/various/nux.spec.js

## Screenshots or screencast

### CODE REVIEW

**Comment by sandeshjangam** [comment]
@kevin940726 Can you please add it in https://github.com/WordPress/gutenberg/issues/38851 ?

Your input is appreciated.

**Comment by kevin940726** [comment]
Is this PR ready for review? I can see that there's still some temporary commented code.

**Comment by sandeshjangam** [comment]
@kevin940726 PR is now ready to review. Removed all unwanted comments and code.

File: test/e2e/playwright.config.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ const config: PlaywrightTestConfig = {
 			strictSelectors: true,
 		},
 		storageState: STORAGE_STATE_PATH,
-		actionTimeout: 10_000, // 10 seconds.
+		actionTimeout: 20_000, // 20 seconds.
```
Comment (kevin940726) [inline test/e2e/playwright.config.ts:40]: Why do we want to change this?

**Review by kevin940726 — COMMENTED** [review]
Thank you for making the PR!

Have you read the [best practices](https://github.com/WordPress/gutenberg/blob/HEAD/test/e2e/README.md#best-practices) guide yet? I noticed that we're still using class names and XPath selectors rather than using accessible role selectors in the tests. (We could refactor them in the future though, but I'd prefer to refactor them as we're migrating 🙂)

**Pull request closed without merging**